external-builds/pytorch/README.md

```diff
@@ -35,7 +35,7 @@ This incorporates advice from:
 | torch                          | ✅ Supported                                                                                                                  | ✅ Supported    |
 | torchaudio                     | ✅ Supported                                                                                                                  | ✅ Supported    |
 | torchvision                    | ✅ Supported                                                                                                                  | ✅ Supported    |
-| Flash attention via [ao]triton | ✅ Supported for torch < 2.10<br>❌ Disabled for torch ≥ 2.10 (see [Issue#1408](https://github.com/ROCm/TheRock/issues/1408)) | ✅ Supported    |
+| Flash attention via [ao]triton | ✅ Supported for torch < 2.9<br>❌ Disabled for torch ≥ 2.9 (see [Issue#1408](https://github.com/ROCm/TheRock/issues/1408)) | ✅ Supported    |
```

Comment:
This is tricky. We previously built torch 2.8 (and 2.9?) on Linux with aotriton enabled, but we can't build _new_ versions of those packages currently due to how changes have been backported.

The docs here are a mix of user-facing (historical) and developer-facing (latest).